packages/cms-cli/lib/commonOpts.js

```diff
@@ -24,6 +24,10 @@ const addOverwriteOptions = program => {
   program.option('--overwrite', 'overwrite existing files', false);
 };
 
+const addTestingOptions = program => {
+  program.option('--qa', 'use QA environment for command', false);
+};
```

Comment:
Can we suppress this from the help since it is only relevant for internal testing?

Reply:
I don't see a way in commander to supress any registered option. If I leave this off, the necessary data will not get passed along in the options data so the flag won't work.